semgrep-core/parsing/Parse_kotlin_tree_sitter.ml

```diff
@@ -73,10 +73,10 @@ let pat_b294348 (env : env) (tok : CST.pat_b294348) =
 
 let visibility_modifier (env : env) (x : CST.visibility_modifier) =
   (match x with
-   | `Public tok -> token env tok (* "public" *)
-   | `Priv tok -> token env tok (* "private" *)
-   | `Inte tok -> token env tok (* "internal" *)
-   | `Prot tok -> token env tok (* "protected" *)
+   | `Public tok -> KeywordAttr (Public, token env tok) (* "public" *)
+   | `Priv tok -> KeywordAttr (Private, token env tok) (* "private" *)
+   | `Inte tok -> NamedAttr (token env tok, [str env tok], empty_id_info (), fake_bracket []) (* "internal" *)
```

Comment:
maybe factorize in an helper function like 'unhandled_keyword_attr_to_named_attr tok'

Reply:
good idea!